packages/dream/dream.1.0.0~alpha1/opam

```diff
@@ -58,8 +58,9 @@ depends: [
   "graphql-lwt"
   "hmap"
   "lwt"
-  "lwt_ppx"
+  "lwt_ppx" {>= "1.2.2"}
```

Comment:
Before I cherry-pick this into Dream, I want to ask: I'm under the impression that Dream should build even with `lwt_ppx` < 1.2.2. Should it deliberately exclude earlier versions in order to enforce Dune? Did it fail to build?

Reply:
full log: https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/000177c2a47359c71a41e9fc78d084954efdf708/variant/opam-2.1,compilers,4.08,dream.1.0.0~alpha2,lower-bounds